UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/FileContentHelper.cs

```diff
@@ -39,22 +42,23 @@ public static async Task<List<string>> DownloadAndExtractExchangeSetZipFileForLa
             List<string> downloadFolderPath = new();
 
             string mediaType = batchDetailsResponse.attributes[1].value;
-
+            
             if (mediaType.Equals("Zip"))
             {
                 string fileName = batchDetailsResponse.files[0].filename;
-                if (fileName.Equals($"{posDetails.UpdateExchangeSet}"))
+                if (fileName.Equals(string.Format(posDetails.PosUpdateZipFileName, weekNumber, currentYear)))
                 {
-                    string downloadFileUrl = $"{Config.FssConfig.BaseUrl}/batch/{batchId}/files/{posDetails.UpdateExchangeSet}";
+                    string zipFileName = string.Format(posDetails.PosUpdateZipFileName, weekNumber, currentYear);
```

Comment:
Can we use string interpolation please in this whole class wherever feasible.

Reply:
Not possible because we are altering the filename after collecting it from appsettings.